src/main/java/com/openshift/internal/restclient/model/ResourceRequirements.java

```diff
@@ -0,0 +1,99 @@
+/*******************************************************************************
+ * Copyright (c) 2017 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.openshift.internal.restclient.model;
+
+import java.util.Optional;
+
+import org.jboss.dmr.ModelNode;
+
+import com.openshift.restclient.model.IResourceRequirements;
+import com.openshift.restclient.model.IResourceSpec;
+
+/**
+ * @author Jeff Maury
+ *
+ */
+public class ResourceRequirements implements IResourceRequirements {
```

Comment:
@jeffmaury sorry for the delay... apparently my review comments were not submitted